specification/compute/resource-manager/readme.md

```diff
@@ -27,16 +27,53 @@ To see additional help and options, run:
 
 
 
-### Basic Information 
+### Basic Information
 These are the global settings for the Compute API.
 
 ``` yaml
 title: ComputeManagementClient
 description: Compute Client
 openapi-type: arm
-tag: package-2017-03
+tag: package-2017-09
+```
+
+### Tag: package-2017-09
+
+These settings apply only when `--tag=package-2017-09` is specified on the command line.
+
+``` yaml $(tag) == 'package-2017-09'
+input-file:
+- Microsoft.Compute/2017-03-30/compute.json
+- Microsoft.Compute/2017-03-30/disk.json
+- Microsoft.Compute/2017-03-30/runCommands.json
+- Microsoft.ContainerService/2017-07-01/containerService.json

```

Comment:
@mboersma Any particular reason why the ACS/AKS does not have its own resource manager and it's still part of the compute resource manager?

Reply:
@milismsft I have no idea. Maybe @rjtsdl or @shrutir25 have some historical perspective to share on this? I've only been here a few months. 😄

Reply:
I don't know the historical reason either... @mboersma , i have only been here exact a year :)

Reply:
Note that Python ships this in a different package for 6 months now (it's why the `package-container-service-2017-09` tag is important). And I think Ruby is going to do the same at least.
Shipping this with Compute is a nightmare for multiapi versionning (and then Stack support), since ACS has nothing to do in term of RP and apiversion with Compute...

Reply:
@lmazuel I totally understand, it seemed odd to me as well. I think we have both assumed there were good existing reasons for compute and containerservices cohabitating.

Maybe we could look at moving it under `specification/containerservices` at some point? We do have some API changes in the pipeline, and this will probably be an active development area for a while. I'm just not aware of everything that would break with such a change.

Reply:
Moving it to the specification/containerservice will be great. I generated the Java SDK by pointing Autorest at package-container-service-2017-09 and had to overwrite the default "title" which currently defaults to "ComputeManagementClient" in order to avoid any confusion as how the default container service manager is named; having its own readme.md will help avoid such workarounds.

Reply:
Same as @milismsft :):
https://github.com/Azure/azure-sdk-for-python/blob/67e570d79d767edb770a304cec23bf5ee2c4efc6/swagger_to_sdk_config.json#L136-L139

Big +++ to split that into another folder.

Reply:
Just to follow up, we confirmed the reason for ACS living in the Compute spec is purely historical. The Container Services group started within Compute and just started the new API spec there.